### PR TITLE
Add zh-hk translations

### PR DESCRIPTION
**Pull request by swonge**

### CODE REVIEW

File: src/localize/languages/zh-hk.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "1": "星期一",
+      "2": "星期二",
+      "3": "星期三",
+      "4": "星期四",
+      "5": "星期五",
+      "6": "星期六",
+      "7": "星期日"
+    },
+    "misc": {
+      "aqi": "AQI",
```
Comment (pkissling) [inline src/localize/languages/zh-hk.json:29]: any reason not to translate this?

File: src/localize/languages/zh-hk.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "humidity": "濕度",
+      "feels-like": "Feels like"
```
Comment (pkissling) [inline src/localize/languages/zh-hk.json:31]: same here

Comment (swonge) [reply src/localize/languages/zh-hk.json:31]: Sorry, I missed that, will make change on that. Good catch! Thx

**Comment by pkissling** [comment]
closing orphaned pull request

**Pull request closed without merging**